source/SpinalHDL/Data types/bits.rst

```diff
@@ -92,29 +101,41 @@ Logic
    * - x.andR
      - AND all bits of x
      - Bool
-   * - x \>\> y
-     - Logical shift right, y: Int
+   * - | y = 1 // Int
```

Comment:
I still think that it's easier to read without the duplicated information.

Reply:
I think you are citing the TYPE info column as duplicate information ?  Maybe to you as an advanced user, but to a novice that is type information (like the heading says - I'm not suggesting a change here BTW).

It is much better to restate things as key-points in clear readable language than use a new foreign cryptic notation to the reader, that add additional load to the reader trying to decode/understand what is being read.

Maybe the right hand column would work well with a tooltip style hover over, explaining the meaning of the notation in plain language.

`The result is the same data type as the original at the same width.`

It would however remove it from the page for any PDF/screen-capture.